tools/c7n_azure/tests_azure/container-host/test_container_host.sh

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env bash
+
+set -e
+export AZURE_CORE_OUTPUT="none"
+
+resource_group_name="custodian-container-host-nightly"
```

Comment:
Can you add some randomness so we can run it concurrently? (You can add `$RANDOM` at the end)

Reply:
We can now run the script concurrently. It took some other tweaks to the policy upload, kubectl, and helm stuff too.